flux_local/tool/diff.py

```diff
@@ -225,9 +234,10 @@ def create_diff_path(
         yield git_repo.PathSelector(path_orig, sources=kwargs.get("sources"))
         return
 
-    with git_repo.create_worktree(selector.repo) as worktree:
-        yield git_repo.PathSelector(pathlib.Path(worktree) / selector.relative_path)
+    branch_orig = kwargs.get('branch_orig', None)
```

Comment:
The `None` is not needed here as it is already the default value so prefer to write without it. While it makes it more explicit, this is a common enough paradigm that it its OK to be implicit.

```suggestion
    branch_orig = kwargs.get('branch_orig')
```

Reply:
Consider inlining the whole thing in the `create_worktree` branch if it can fit since its a trivial expression and the variable isn't used again.